db/people.yml

```diff
@@ -5338,26 +5338,23 @@ swhan:
 swu:
   active: true
   aka: []
-  avatar: "https://github.com/\x16SeanWu25.png"
-  bio: Sean is a third year undergraduate student at Pepperdine University. He is
-    studying Computer Science/Mathematics and is interested in advancing medicine with machine learning.
-    He loves to play basketball and piano. He is conducting research with the Columbia SURE
-    program for Summer 2023.
-  education: [Pepperdine]
-  email: "\x16sean.wu@pepperdine.edu"
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: 'Sean is a third year undergraduate student at Pepperdine University. He is studying Computer Science/Mathematics and is interested in advancing medicine with machine learning. He loves to play basketball and piano. He is conducting research with the Columbia SURE program for Summer 2023.'
+  education:
+    - begin_year: 2023
+      institution: pepperdine university
+      degree: visiting student
```

Comment:
probably your degree is something like BA, BSc something ike that.  Please look at other entries to get the right format.

Please add begin_month and you can put end_month and end_year in the future.

Reply:
On it right now